# Undertanding the cartwheel data set

The notebook aims to undertand the content of the cartwheel data set.


## Acknowledgments

- Data from https://www.coursera.org/ from the course "Understanding and Visualizing Data with Python" by University of Michigan


# Cartwheel data set

1. A cartwheel

![cartwheel1.png](datasets/cartwheel/cartwheel1.png)

2. The dataset description
    - The dataset used here is an extension from the original cartwheel dataset from cursera
    - Total numer of observations: 28
    - Many observations/measurements/recordings of the characteristics/attributes/variables of cartwheel executions
    - Variables: Age, Gender, GenderGroup, Glasses, GlassesGroup, Height, Wingspan, CWDistance, ... (X variables)
    


# Importing and inspecting the data

In [16]:
# Import the packages that we will be using
#import matplotlib.pyplot as plt
import pandas as pd                 # For data handling

# Dataset url
url = "datasets/iris/iris.csv"

# Load the dataset
dataset = pd.read_csv(url)


In [17]:
dataset

SepalLength  SepalWidth  PetalLength  PetalWidth           Class
0            5.1         3.5          1.4         0.2     Iris-setosa
1            4.9         3.0          1.4         0.2     Iris-setosa
2            4.7         3.2          1.3         0.2     Iris-setosa
3            4.6         3.1          1.5         0.2     Iris-setosa
4            5.0         3.6          1.4         0.2     Iris-setosa
..           ...         ...          ...         ...             ...
145          6.7         3.0          5.2         2.3  Iris-virginica
146          6.3         2.5          5.0         1.9  Iris-virginica
147          6.5         3.0          5.2         2.0  Iris-virginica
148          6.2         3.4          5.4         2.3  Iris-virginica
149          5.9         3.0          5.1         1.8  Iris-virginica

[150 rows x 5 columns]

# Final remarks


- The **Pandas** library provides fancy, high-performance, easy-to-use data structures and data analysis tools


# Activity: work with the iris dataset

1. Load the iris.csv file in your computer and compare with the data printed above.


2. How many observations (rows) are in total?
150 observations

3. How many variables (columns) are in total? What do they represent?
5 variables

4. How many observatoins are for each type of flower? 
50 observations for each type of flower

5. What is the type of data for each variable? 
SepalLength = float
SepalWidth  =  float
PetalLength  =  float
PetalWidth = float
Class = string

6. What are the units of each variable?
SepalLength = cm
SepalWidth  =  cm
PetalLength  =  cm
PetalWidth = cm
Class = categoria/clasificacion